docs/userguide/icu4c-readme.md

```diff
@@ -0,0 +1,1030 @@
+---
+layout: default
+title: ICU4C Readme
+nav_order: 8
+parent: ICU
+---
+<!--
+© 2020 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# ICU4C Readme
+{: .no_toc }
+
+## Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+
+**Note:** This is a draft readme.
```

Comment:
Ok. Just a question: Is this the entire contents of the ICU4C readme, just partially converted to Markdown?

Reply:
Yes, that's exactly right.  It was taking some time to convert some of the content to Markdown, so I only got partially done with it.  Some parts were not possible to convert to Markdown, so they remained as HTML.